pkg/k8s/diverts/divert.go

```diff
@@ -68,46 +70,44 @@ func divertService(ctx context.Context, m *model.Manifest, name string, c kubern
 		if !oktetoErrors.IsNotFound(err) {
 			return err
 		}
-		s = translateService(m, from, "")
+		s = translateService(m, from)
 		if _, err := c.CoreV1().Services(m.Namespace).Create(ctx, s, metav1.CreateOptions{}); err != nil {
-			return err
+			if !oktetoErrors.IsAlreadyExists(err) {
+				return err
+			}
 		}
-		return divertEndpoints(ctx, m, name, c)
+		return divertEndpoints(ctx, m, from, c)
 	}
 
 	if s.Annotations[model.OktetoAutoCreateAnnotation] != "true" {
 		return nil
 	}
 
-	s = translateService(m, from, s.ResourceVersion)
+	s = translateService(m, from)
```

Comment:
I removed the `resourceVersion` check everywhere. Otherwise,  `okteto deploy` commands running in parallel might fail. Since they create the same yaml, it's safe to ignore this error.
At some point, all this logic should be handled by an operator, but that requires a major refactor